snap/local/etc/cruise-control/log4j.properties

```
@@ -0,0 +1,60 @@
+#
+# Copyright 2017 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License"). See License in the project root for license information.
+#
+
+cruisecontrol.log.level=INFO
+
+rootLogger.level=${sys:cruisecontrol.log.level}
+appenders=kafkaCruiseControlAppender, operationAppender, requestAppender
+
+property.filename=${sys:cruisecontrol.logs.dir}
+
+appender.kafkaCruiseControlAppender.type=RollingFile
+appender.kafkaCruiseControlAppender.name=kafkaCruiseControlFile
+appender.kafkaCruiseControlAppender.fileName=${filename}/kafkacruisecontrol.log
+appender.kafkaCruiseControlAppender.filePattern=${filename}/kafkacruisecontrol.log.%d{yyyy-MM-dd-HH}
+appender.kafkaCruiseControlAppender.layout.type=PatternLayout
+appender.kafkaCruiseControlAppender.layout.pattern=[%d] %p %m (%c)%n
+appender.kafkaCruiseControlAppender.policies.type=Policies
+appender.kafkaCruiseControlAppender.policies.time.type=TimeBasedTriggeringPolicy
+appender.kafkaCruiseControlAppender.policies.time.interval=1
```

I suggest to keep the same approach as we recently did with Kafka:
```suggestion
appender.kafkaCruiseControlAppender.policies.size.type = SizeBasedTriggeringPolicy
appender.kafkaCruiseControlAppender.policies.size.size=100MB
appender.kafkaCruiseControlAppender.strategy.type = DefaultRolloverStrategy
appender.kafkaCruiseControlAppender.strategy.max = 10
```

---

After checking, there are up to 6 files between kafka and cc. A fully running cluster log output might be too much if each unit is going to have up to 3Gb of logs.